src/modules/Accordion/Accordion.js

```diff
@@ -38,6 +45,9 @@ export default class Accordion extends Component {
     /** Initial activeIndex value. */
     defaultActiveIndex: PropTypes.number,
 
+    /** Allow only one index */
+    exclusive: PropTypes.bool,
```

Comment:
We try to stick very close to SUI core descriptions.  Here's the description from the [Accordion settings](http://semantic-ui.com/modules/accordion#/settings) page:

```js
/** Only allow one section open at a time */
```

Since we currently have a deviation in terms, `panels` vs `sections`, let's keep consistent with `panels`.  Suggest we go with:

```js
/** Only allow one panel open at a time */
```